zerver/lib/send_email.py

```diff
@@ -542,17 +542,20 @@ def send_custom_email(users: List[UserProfile], options: Dict[str, Any]) -> None
             "realm_name": user_profile.realm.name,
             "unsubscribe_link": one_click_unsubscribe_link(user_profile, "marketing"),
         }
-        send_email(
-            email_id,
-            to_user_ids=[user_profile.id],
-            from_address=FromAddress.SUPPORT,
-            reply_to_email=options.get("reply_to"),
-            from_name=get_header(
-                options.get("from_name"), parsed_email_template.get("from"), "from_name"
-            ),
-            context=context,
-            dry_run=options["dry_run"],
-        )
+        try:
+            send_email(
+                email_id,
+                to_user_ids=[user_profile.id],
+                from_address=FromAddress.SUPPORT,
+                reply_to_email=options.get("reply_to"),
+                from_name=get_header(
+                    options.get("from_name"), parsed_email_template.get("from"), "from_name"
+                ),
+                context=context,
+                dry_run=options["dry_run"],
+            )
+        except EmailNotDeliveredException:
+            pass
```

Comment:
We probably want to log something when this happens, right?

Reply:
This will already have logged, I believe.  Let me verify.

Reply:
Yes, this already logs:
```
2021-12-14 23:30:18.700 INFO [zulip.send_email] Sending custom_email_d2dc93dea2f55ed46b194a5d19f950e8 email to ['...snip... <example@example.com>']
2021-12-14 23:30:18.874 ERR  [zulip.send_email] Error sending custom_email_d2dc93dea2f55ed46b194a5d19f950e8 email to ['...snip... <example@example.com>'] with error core 554: b"Transaction failed: Address length is more than 320 bytes long: '...snip... <example@example.com>'."

Reply:
Great, thanks for confirming!